components/Header.jsx

```diff
@@ -0,0 +1,77 @@
+import Link from "next/link";
+import styled from "styled-components";
+import LibrarySvg from "@/public/library.svg";
+import ListSvg from "@/public/list.svg";
+import PlusSvg from "./PlusSvg";
+import { useRouter } from "next/router";
+
+export default function Header() {
+  const router = useRouter();
+
+  return (
+    <StyledNavigation>
+      <StyledList>
+        <li>
+          <StyledNavLink
+            href="/library"
+            aria-label="view library"
+            $active={router.pathname === "/library"}
+          >
+            <LibrarySvg />
+            <p>Library</p>
+          </StyledNavLink>
+        </li>
+        <li>
+          <StyledNavLink
+            href="/add-game"
+            aria-label="add game"
+            $active={router.pathname === "/add-game"}
+          >
+            <PlusSvg width="50" height="50" />
+            <p>Add Game</p>
+          </StyledNavLink>
+        </li>
+        <li>
+          <StyledNavLink
+            href="/"
+            aria-label="view personal list"
+            $active={router.pathname === "/"}
+          >
+            <ListSvg />
+            <p>List</p>
+          </StyledNavLink>
+        </li>
+      </StyledList>
+    </StyledNavigation>
+  );
+}
+
+const StyledNavigation = styled.nav`
+  position: fixed;
+  left: 0px;
+  top: 0px;
+  width: 100%;
+  padding: 0.5em;
+  background-color: var(--backgroundSubSection);
+  display: none;
+
+  @media screen and (min-width: 1024px) {
+    display: block;
+  }
+`;
+
+const StyledList = styled.ul`
+  list-style: none;
+  display: flex;
+  justify-content: flex-end;
+  gap: 3em;
+  margin-right: calc((100% - 1144px) / 2);
```

Comment:
I noticed a bug when trying out different screen sizes for your application.

When I am on any page and my screen width is between 1024 and around 1160px, I am not able to see the navigation completely. The link to my game list is cropped off.

And I think this line of code is responsible. If you changed it to:
```suggestion
  margin-right: calc((100% - 1024px) / 2);
```

The navigation would no longer be cropped off, but neither would it move more to the center as you had it before.

So you might want to try out which behaviour you want as a team.